src/styles/stopWatch.scss

```diff
@@ -0,0 +1,53 @@
+.container {
+  width: 100vw;
+  height: 100vh;
+
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.stopWatch {
+  position: relative;
+
+  width: 50%;
+  height: 50%;
```

Comment:
The stopwatch container is too small
<img width="786" alt="Screenshot 2023-03-23 at 12 43 51" src="https://user-images.githubusercontent.com/23585899/227193725-7ba365f2-7c0d-4256-a7a4-efb3b982c705.png">